EIPS/eip-addFromField.md

```diff
@@ -0,0 +1,48 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Separating transaction signer from transaction deployer
+    Author: Alex Van de Sande
+    Type: ERC
+    Status: Draft
+    Created: 2018-02-09
+
+
+## Simple Summary
+
+Many contract developers run in the problem of allowing users to interact with their contracts without having ether. This has been addressed by proposed [account abstractions](https://github.com/ethereum/EIPs/issues/859) which would bring a lot of new features but also brings a lot of complexity. The latest proposed version by Vitalik, while still brings many benefits like quantum resistance, would still not, for example, to pay token transactions with the token themselves, an often requested feature.
+
+This EIP proposes a simple way that enables this by simply separating the transaction ***signer*** (the person or entity authorizing the transaction) from the transacion ***deployer*** (the person or entity publishing that transaction to the chain and paying its gas). It is not meant to replace account abstraction and the other benefits it might bring.
+
+## Abstract
+
+Currently, ethereum transactions have the following fields: `nonce`, `gasPrice`, `gasLimit`, `value`, `to` and `data`. This EIP proposes adding a new one: `from` which, if present, indicates that `data` is an ethereum transaction containing `to`, `value`, `to` and its own `data`, but the message is signed by the account represented on `from`.
```

Comment:
> `to`, `value`, `to` and its own `data`,

Looks like one `to` many.

Reply:
Maybe it's worth emphasizing that the inner transaction does not have `nonce` field.